content/manuals/desktop/settings.md

```diff
@@ -258,8 +258,10 @@ The HTTPS proxy settings used for scanning images are set using the `HTTPS_PROXY
 > If you are using a PAC file hosted on a web server, make sure to add the MIME type `application/x-ns-proxy-autoconfig` for the `.pac` file extension on the server or website. Without this configuration, the PAC file may not be parsed correctly.
 
 > [!IMPORTANT]
->
-> You do not need to separately configure proxy settings for the Docker CLI or Docker daemon.
+> You cannot separately configure proxy settings for Docker Desktop using the
+> Docker CLI or Docker daemon configuration files. Proxy configurations for
```

Comment:
CLI configuration should be supported I think? Because those configure the proxy for the container, or do we not allow those to be set as well? (that'd be .. not correct if we don't)

Reply:
I think if you know what you're doing you could set the container proxy via the CLI (or environment variables in the container) but it would probably only work if
- the proxy is a constant value (sounds obvious, but in come environments the correct proxy varies per-URL so a single setting won't work)
- the proxy supports basic auth (instead of e.g. Kerberos)
- the container code respects the proxy environment vars (sounds obvious, but I've written lots of bad code which doesn't!)

I think it's more likely to work in DD in corp environments if the container connects directly and DD transparently proxies the traffic.

Perhaps we should separate the daemon and CLI:
- "You cannot configure the proxy settings using the Docker daemon configuration file" <- this is true because we block that
- "and we recommend you do not configure the proxy settings via the Docker CLI configuration file" <- a recommendation because it's more likely to "just work"

WDYT?

Reply:
@thaJeztah my understanding too was that this specific change would not have an impact on proxy in config.json. Tested with proxy in config.json and verified it is correct, proxy set in config.json is still honored after this change.

I agree with @djs55 suggestions that for Docker CLI we can mention it as a recommendation.

Reply:
Hm, right, so the CLI configuration was originally _on purpose_ separate from the daemon configuration; i.e., while the daemon may need (or desired to be using) a proxy, that configuration is _separate_ from what the container uses. (container in this case; containers used for both `docker run` as well as `docker build`). 

I guess the situation would be different if Docker Desktop transparently proxies traffic for _any_ networking happening from within the VM (both daemon and containers). The only thing I'm wondering here is; I recall situations where people indicated it was _not desirable_ for containers to automatically get the proxy configured, and I wonder if that's still a use-case we can satisfy if the VM automatically provides it.

Reply:
Good point about a different config from containers -- in DD it's now possible to have separate proxy configuration for the (traffic from) containers as part of the ["air gapped containers" feature](https://docs.docker.com/security/for-admins/hardened-desktop/air-gapped-containers/) so hopefully that's enough 🤞